src/page/shipping-step.page.ts

```diff
@@ -0,0 +1,17 @@
+import { $, ElementFinder, browser } from 'protractor';
+
+export class ShippingStepPage {
+  private acceptTerms: ElementFinder;
+  private checkOut: ElementFinder;
+
+  constructor() {
+    this.acceptTerms = $('#cgv');
+    this.checkOut = $('#form > p > button > span');
+  }
+
+  public async acceptanceTermsService(): Promise<void> {
```

Comment:
as you are performing two actions here, you can name  this method in a better way, like: acceptTermsAndContinue or acceptToSAndCheckout